src/mol-util/zip/util.ts

```diff
@@ -1,12 +1,25 @@
 /**
- * Copyright (c) 2020 mol* contributors, licensed under MIT, See LICENSE file for more info.
+ * Copyright (c) 2020-2024 mol* contributors, licensed under MIT, See LICENSE file for more info.
  *
  * @author Alexander Rose <alexander.rose@weirdbyte.de>
  *
  * ported from https://github.com/photopea/UZIP.js/blob/master/UZIP.js
  * MIT License, Copyright (c) 2018 Photopea
  */
 
+let hasCompressionStreamSupport: boolean | undefined = undefined;
+export function checkCompressionStreamSupport(format: 'deflate' | 'deflate-raw' | 'gzip'): boolean {
+    if (hasCompressionStreamSupport === undefined) {
+        try {
+            new CompressionStream(format);
```

Comment:
Could only some formats be supported? Because if so, there is a bug.

Reply:
ouch